Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -1047,47 +1047,53 @@ private void InitialiseInterfaces()
                             // populate interface address list
                             foreach (UnicastIPAddressInformation info in ipProperties.UnicastAddresses)
                             {
-                                if (IsIP4Enabled && info.Address.AddressFamily == AddressFamily.InterNetwork)
+                                if (info.Address.AddressFamily == AddressFamily.InterNetwork)
```

Comment:
Looks like all of this can be reverted as well, right?

Reply:
done